nexus/db-queries/src/db/datastore/ip_pool.rs

```diff
@@ -108,13 +108,10 @@ impl DataStore {
             ),
         }
         .inner_join(ip_pool_resource::table)
-        .filter(
-            ip_pool_resource::resource_type
-                .eq(IpPoolResourceType::Silo)
-                .and(ip_pool_resource::resource_id.eq(silo_id)),
-        )
+        .filter(ip_pool_resource::resource_type.eq(IpPoolResourceType::Silo))
+        .filter(ip_pool_resource::resource_id.eq(silo_id))
         .filter(ip_pool::time_deleted.is_null())
-        .select(db::model::IpPool::as_select())
+        .select(<(IpPool, IpPoolResource)>::as_select())
```

Comment:
neat trick. initially I tried `(IpPool::as_select(), IpPoolResource::as_select())` but it wanted the above